apps/web/src/components/Button/ButtonWithLinkAndEventLogging.tsx

```diff
@@ -0,0 +1,26 @@
+'use client';
+
+import { useCallback } from 'react';
+
+import { ActionType } from 'libs/base-ui/utils/logEvent';
+
+import { useAnalytics } from 'apps/web/contexts/Analytics';
+
+import { ButtonWithLink, ButtonWithLinkProps } from './ButtonWithLink';
```

Comment:
nitpick: no need to add space between each import

Reply:
all set, removed unnecessary spaces from all files